### ISSUE TITLE
[Bug Report] Question: Im getting large jitter frames

### ISSUE THREAD

**Issue by DaemonGenius** [jammsen/docker-palworld-dedicated-server#84]
### Have you read the Important information text above

- [X] Yes i did

### Describe the feature

![image](https://github.com/jammsen/docker-palworld-dedicated-server/assets/17024042/5de17bd2-281f-4e73-b393-0ac5066be178)

I'm getting large jitter frames and some of the pals are teleporting. Want to know if this could be a resource issue? I hosting it locally on the same machine I'm playing on, planning on moving it over to my server later today. 

### Additional information

- [ ] Would you be willing to help implement this feature?

### Final checks

- [X] I understand that this is a feature request function. I also unterstand that if i use this to report a bug report, that my issue might be un-answered closed.

**Comment by jammsen** [comment]
Hey @DaemonGenius you could try the just added feature ( https://github.com/jammsen/docker-palworld-dedicated-server/commit/410d2e7ab7be870a197bc53c9badb05a76975d3b )
In the documentation there are examples for changing the Tick Rate, MAYBE that could help?

**Comment by jammsen** [comment]
Please use the bug issue type next time for a question, that way i at least know your hardware and software, now im just quessing blindly with a touch of maybe 🤣

**Comment by DaemonGenius** [comment]
Ohh okay, didn't see that. Awesome. 

You guys are moving at light speed. Thanks so much. 


Yeah was doing a Bug Report but then I felt like "ehhh prob not a bug" lol

Will play around and give feedback.

**Comment by jammsen** [comment]
> Ohh okay, didn't see that. Awesome.

Lol its not that old yet 👍 

> You guys are moving at light speed. Thanks so much.

Im mostly alone, but i try my best!
> Yeah was doing a Bug Report but then I felt like "ehhh prob not a bug" lol

LOL

> Will play around and give feedback.

Waiting for it!

**Comment by Damien13150** [comment]
Hey ! Experiencing the same issue here. Changing the tickrate didn't solve it

**Comment by DaemonGenius** [comment]
When I moved it over to my server my issues were solved. Currently have 8 Pals in base and the game is smooth with around 5gig of RAM usage. 

Technical Specifications
Product: Dell OptiPlex 3080
Processor frequency: 2.3 GHz
Processor family: 10th gen Intel® Core™ i5
Processor model: i5-10500T
Internal memory: 24 GB
OS: Linux

I was testing on my Windows machine earlier so not sure if it's the WSL 2 docker engine that might cause it.
